focus-android/app/src/main/res/values/strings.xml

```diff
@@ -696,11 +696,11 @@
     <string name="cookie_banner_exception_panel_description_state_off_for_site">%1$s can try to automatically reject cookie requests.</string>
 
     <!-- Contextual Feature Recommendation Popups -->
-    <!-- CFR for Cookie Banner (Banner Info Message). The placeholder will be updated with the app name. -->
-    <string name="cfr_for_cookie_banner">%1$s tries to reject cookie requests to dismiss annoying cookie banners.\n\nManage cookie banner preferences in settings.</string>
+    <!-- CFR for Cookie Banner (Banner Info Message). %1$s will be replaced by the app name, %2$s will be an active link using the string cfr_cookie_banner_link as text. -->
+    <string name="cfr_cookie_banner">%1$s tries to reject cookie requests to dismiss annoying cookie banners.\n\nManage cookie banner preferences in %2$s.</string>
```

Comment:
@iorgamgabriel Is safe to update already existing strings? In Fenix instead of updating we re-create the string with a new ID and mark it for future deletion as it could cause crash see https://github.com/mozilla-mobile/firefox-android/blob/main/focus-android/docs/Removing-strings.md

Reply:
Done